src/services/sanity.js

```diff
@@ -534,6 +535,7 @@ export async function fetchAll(brand, type, {
         const lessonsFilter = `brand == '${brand}' && ^._id in ${groupBy}[]._ref ${typeFilter} ${searchFilter} ${includedFieldsFilter} ${progressFilter}`;
         entityFieldsString = `
                 'id': _id,
+                railcontent_id,
```

Comment:
In this case (grouped by genre), the railcontent_id will always be null.

Reply:
In some cases (workouts, for example), group by instructor is using this case, so railcontent_id is the instructor's railcontent_id. 🤔

Reply:
Yes, when we have group by instructor the railcontent_id will be set.